2024-04-05 17:14:28 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de8d00>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:14:28 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de8bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:14:28 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de8640>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:14:28 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de9cc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:14:28 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de9e40>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:14:28 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x107de9ff0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106de8bb0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106de8850>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106de8550>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106de9c90>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106de9e10>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:18:34 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x106d9a8f0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179e8850>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179e88b0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179e8640>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179e9cc0>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x117997b50>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
2024-04-05 17:20:41 WARNING Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NameResolutionError("<urllib3.connection.HTTPSConnection object at 0x1179e8490>: Failed to resolve 'o151352.ingest.sentry.io' ([Errno 8] nodename nor servname provided, or not known)")': /api/4504800232407040/envelope/
